go.mod

```
@@ -207,7 +208,7 @@ replace github.com/dgrijalva/jwt-go => github.com/golang-jwt/jwt/v4 v4.4.2
 replace github.com/rjeczalik/notify => github.com/rjeczalik/notify v0.9.3
 
 // use cometbft
-replace github.com/tendermint/tendermint => github.com/cometbft/cometbft v0.34.31
+replace github.com/tendermint/tendermint => github.com/cometbft/cometbft v0.34.34
```

```suggestion
replace github.com/tendermint/tendermint => github.com/cometbft/cometbft v0.34.35
```
looks like the latest is v0.34.35